src/benchmarks/micro/corefx/System.Buffers/ArrayPoolTests.cs

```diff
@@ -152,6 +153,7 @@ public async Task ProducerConsumer()
     [BenchmarkCategory(Categories.CoreFX)]
     [GenericTypeArguments(typeof(byte))] // value type
     [GenericTypeArguments(typeof(object))] // reference type
+    [GcServer(true)] // Server GC is the most important scenario for ArrayPool
```

Comment:
Is doing this per test suite the right thing to do?  Or would it be better to do GcServer(false) runs for all benchmarks and GcServer(true) runs for all benchmarks?

I imagine that many of our benchmarks are allocation-heavy and would be influenced by client vs server GC.

Reply:
I would hope the vast majority of our benchmarks do not allocate, or allocate very little. I certainly don't want to duplicate the matrix on this pivot without strong evidence it makes a big difference and is useful. My guess is that doing this experiment to highlight any unexpected surprises once and track those as individual work items is the better strategy.

Reply:
We have very few allocation-heavy benchmarks. I think that what we do now (don't enforce Server GC) is the best thing.